setup.py

```diff
@@ -76,3 +93,5 @@ def package_files(directory):
         "Topic :: Scientific/Engineering :: Electronic Design Automation (EDA)",
     ],
 )
+
+print(log_stream.getvalue())
```

Comment:
Perhaps
```suggestion
sys.stderr.write(log_stream.getvalue())
```

Reply:
Is perhaps nicer but is this is not an error?

Reply:
Matters only if someone is piping the output of `pip`, which probably doesn't matter anyway

Reply:
Also avoids an extra trailing newline